### DISCUSSION TITLE
Download Openvpn.conf file from Fritzbox

### DISCUSSION THREAD

**Discussion by fskander** [Freetz-NG/freetz-ng#354] [Allgemein]
Hello,

I installed freetz on my Fritzbox 7530 and I am able to access to status/configuration page on the web interface http://fritz.box:81/

I would like to use my Fritzbox as an OpenVPN server. For this I would like to download the generated OpenVPN configuration file : how do I do this ?

I looked it up online, and apparently the file should be on the var/mod/etc directory on the Fritzbox, but I do not know how do I access this directory and download the file to my computer ? I tried over the browser URL (http://fritz.box:81/var/mod/etc) but I guess this is not the right method as I get a 404 error.

Sorry if this is a stupid question, I am a bit of a newbie - it took me many hours of research and trials to install freetz in the first place. Thank you in advance !

**Comment by fda77** [comment]
I think the openvpn "gui v2" is able
